utils/log-target.js

```diff
@@ -22,14 +22,14 @@ import * as logs from './log.js';
 import humanRtv from './rtv/human-rtv.js';
 import releaseChannels from './rtv/release-channels.js';
 
-const CDN_REGEX = new RegExp(
+const AMP_CACHE_REGEX = new RegExp(
```

Comment:
do you think it is more appropriate to call it `GOOGLE_AMP_CACHE_REGEX` to disambiguate from ampjs.org (which is also an amp_cache_regex) and since we also push the "Google Cache" String when we match for it

Reply:
SG, made the change